test_scripts/UnknownEnums/001_unknown_mandatory_param.lua

```diff
@@ -0,0 +1,52 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
I would suggest a few minor updates which will be useful in the future during support of the scripts:
 - remove trailing white spaces (e.g. currently there are few ones in scripts)
 - use the same indentation across all files (e.g. currently there is a mix of 2 and 4 spaces)

Reply:
Addressed in 07b462b